test/integration/PriceTicker.test.ts

```diff
@@ -0,0 +1,209 @@
+        expect(result.cryptoAvailableSupply).toBe(testData.crypto_available_supply);
+        expect(result.cryptoTotalSupply).toBe(testData.crypto_total_supply);
+        expect(result.cryptoMaxSupply).toBe(testData.crypto_max_supply);
+        expect(result.cryptoPercentChange1H).toBe(testData.crypto_percent_change_1h);
+        expect(result.cryptoPercentChange24H).toBe(testData.crypto_percent_change_24h);
+        expect(result.cryptoPercentChange7D).toBe(testData.crypto_percent_change_7d);
+        expect(result.cryptoLastUpdated).toBe(testData.crypto_last_updated);
+        expect(result.cryptoPriceCurrency).toBe(testData.crypto_price_currency);
+        expect(result.crypto24HVolumeCurrency).toBe(testData.crypto_24h_volume_currency);
+        expect(result.cryptoMarketCapCurrency).toBe(testData.crypto_market_cap_currency);
+    });
+
+    test('Should update the PriceTicker', async () => {
+        const PriceTickerModel: PriceTicker = await priceTickerService.update(createdId, testDataUpdated);
+        const result = PriceTickerModel.toJSON();
+
+        expect(result.currency).toBe(testDataUpdated.currency);
+        expect(result.cryptoId).toBe(testDataUpdated.crypto_id);
+        expect(result.cryptoName).toBe(testDataUpdated.crypto_name);
+        expect(result.cryptoSymbol).toBe(testDataUpdated.crypto_symbol);
+        expect(result.cryptoRank).toBe(testDataUpdated.crypto_rank);
+        expect(result.cryptoPriceUsd).toBe(testDataUpdated.crypto_price_usd);
+        expect(result.cryptoPriceBtc).toBe(testDataUpdated.crypto_price_btc);
+        expect(result.crypto24HVolumeUsd).toBe(testDataUpdated.crypto_24h_volume_usd);
+        expect(result.cryptoMarketCapUsd).toBe(testDataUpdated.crypto_market_cap_usd);
+        expect(result.cryptoAvailableSupply).toBe(testDataUpdated.crypto_available_supply);
+        expect(result.cryptoTotalSupply).toBe(testDataUpdated.crypto_total_supply);
+        expect(result.cryptoMaxSupply).toBe(testDataUpdated.crypto_max_supply);
+        expect(result.cryptoPercentChange1H).toBe(testDataUpdated.crypto_percent_change_1h);
+        expect(result.cryptoPercentChange24H).toBe(testDataUpdated.crypto_percent_change_24h);
+        expect(result.cryptoPercentChange7D).toBe(testDataUpdated.crypto_percent_change_7d);
+        expect(result.cryptoLastUpdated).toBe(testDataUpdated.crypto_last_updated);
+        expect(result.cryptoPriceCurrency).toBe(testDataUpdated.crypto_price_currency);
+        expect(result.crypto24HVolumeCurrency).toBe(testDataUpdated.crypto_24h_volume_currency);
+        expect(result.cryptoMarketCapCurrency).toBe(testDataUpdated.crypto_market_cap_currency);
+    });
+
+    test('Should delete the PriceTicker', async () => {
+        expect.assertions(1);
+        await priceTickerService.destroy(createdId);
+        await priceTickerService.findOne(createdId).catch(e =>
+            expect(e).toEqual(new NotFoundException(createdId))
+        );
+    });
```

Comment:
these tests are just mostly automatically generated for the default crud functionalities. 
you had a search function that you implemented and which is quite important for this implementation. yet there's no test for it here? how do you know that it works?

Reply:
I cannot see any tests making sure that the fetching from cmc works as expected and the prices are being updated on the db as expected

Reply:
done